obal/data/roles/build_package/tasks/koji.yml

```diff
@@ -1,25 +1,27 @@
 ---
 - name: 'Set tito_releasers for brew scratch'
   set_fact:
-    build_package_tito_releasers: "{{ releasers | join(' ') | replace('dist-git', 'scratch') }}"
+    build_package_tito_releasers: "{{ releasers | join(' ') | replace('dist-git', 'scratch') | split(' ') }}"
```

Comment:
I don't know if there's a better way to do this.

Reply:
```yaml
{{ releasers | map('replace', 'dist-git', 'scratch') | list }}
```